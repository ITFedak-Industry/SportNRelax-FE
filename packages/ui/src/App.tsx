import { useState } from 'react';

export function App(): JSX.Element {
  const [count, setCount] = useState(0);

  return (
    <>
      <h1 style={{ color: 'red' }}>Vite + React Go fuck yourself</h1>
      <div className="card">
        <button onClick={() => setCount((count) => count + 1)}>
          count is {count}
        </button>
        <p>
          Edit <code>src/App.tsx</code> and save to test HMR
        </p>
      </div>
      <p className="read-the-docs">
        Click on the Vite and React logos to learn more
      </p>
    </>
  );
}
