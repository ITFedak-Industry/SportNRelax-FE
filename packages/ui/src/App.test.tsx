import '@testing-library/jest-dom';
import { render } from '@testing-library/react';

import { App } from './App';

describe('App', () => {
  it('should be defined', () => {
    expect(App).toBeDefined();
  });

  it('should match the snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
