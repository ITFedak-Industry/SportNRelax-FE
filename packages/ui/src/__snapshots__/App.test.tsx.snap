// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<DocumentFragment>
  <h1
    style="color: red;"
  >
    Vite + React Go fuck yourself
  </h1>
  <div
    class="card"
  >
    <button>
      count is 0
    </button>
    <p>
      Edit 
      <code>
        src/App.tsx
      </code>
       and save to test HMR
    </p>
  </div>
  <p
    class="read-the-docs"
  >
    Click on the Vite and React logos to learn more
  </p>
</DocumentFragment>
`;
